packages/smooth_app/lib/knowledge_panel/knowledge_panels/knowledge_panel_page.dart

```diff
@@ -55,69 +52,50 @@ class _KnowledgePanelPageState extends State<KnowledgePanelPage>
   }
 
   @override
-  Widget build(BuildContext context) => Consumer<UpToDateProductProvider>(
-        builder: (
-          final BuildContext context,
-          final UpToDateProductProvider provider,
-          final Widget? child,
-        ) {
-          final Product? refreshedProduct = provider.get(_product);
-          if (refreshedProduct != null) {
-            _product = refreshedProduct;
-          }
-          return SmoothScaffold(
-            appBar: AppBar(
-              title: Text(
-                _getTitle(),
-                maxLines: 2,
-              ),
+  Widget build(BuildContext context) {
+    final LocalDatabase localDatabase = context.watch<LocalDatabase>();
+    _product = localDatabase.upToDate.getLocalUpToDate(_product);
+    return SmoothScaffold(
+      appBar: AppBar(
+        title: Text(_getTitle(), maxLines: 2),
```

Comment:
Could you add the comma?

Reply:
I would prefer not to.
Technically, I can do it.
In that particular case I don't even have a strong opinion for or against that change.
The thing is that if I add a comma, that file will change, and the PR is complicated enough so if I can spare additional things to double-check to my beloved code reviewers I'll be happier.